spec/models/ability_spec.rb

```diff
@@ -172,10 +172,10 @@
     end
   end
 
-  context 'Private read-only Repository' do
-    let(:editor){ create :user } # editor
-    let(:reader){ create :user } # reader
-    let(:item){   create(:repository, access: 'private_r', user: owner) }
+  context 'Private Repository' do
+    let(:editor) { create :user } # editor
+    let(:reader) { create :user } # reader
+    let(:item) { create(:repository, access: 'private_rw', user: owner) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.